packages/configuration-web-recommended/index.json

```diff
@@ -1,9 +1,6 @@
 {
     "connector": {
-        "name": "jsdom",
-        "options": {
-            "waitFor": 5000
-        }
+        "name": "puppeteer"
```

Comment:
Very Nit: I thing this particular change should have its own commit

Reply:
I was debating that but it's 2 breaking changes in a row so might as well get it done in one. I could rename the message though to
>  Breaking: Replace `chrome` with `puppeteer` and make it the default

Reply:
Ok, that could work too :)

Reply:
OK, I'll do it at merge time then :)